src/sonic-yang-models/yang-models/sonic-fine-grained-ecmp.yang

```diff
@@ -48,12 +52,33 @@ module sonic-fine-grained-ecmp {
                  "Fine grained ecmp is used when the nexthop IPs match the FG_NHG_MEMBER IPs";
             }
 
+            enum prefix-based {
+                description
+                    "Fine grained ecmp is used when the route prefix matches the FG_NHG_PREFIX prefix";
+            }
         }
 
         description
             "This enumeration type defines the method used to determine whether fine grained ecmp should be applied.";
     }
+/*
```

Comment:
nit: cleanup comment since its not applicable anymore?

Reply:
Removed